cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/a_check.spec.js

```diff
@@ -3,11 +3,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
-import { WORKSPACE_NAME, DATASOURCE_NAME } from './constants';
+import { WORKSPACE_NAME, DATASOURCE_NAME } from '../../../../../utils/apps/constants';
```

Comment:
should this be `../utils/apps/query_enhancements/constants`?

Reply:
I put `export * from './query_enhancements/constants';` in the apps constants. Should be good.